client-libraries/java/rest-client/src/com/google/android/gcm/server/Message.java

```diff
@@ -128,6 +131,14 @@ public Builder restrictedPackageName(String value) {
       return this;
     }
 
+    /**
+     * Sets the priority property.
+     */
+    public Builder priority(String value) {
```

Comment:
Given that there are only 2 allowed values for priority, "normal" and "high", you could create an enum and then inside the builder switch on the value and convert to the switch to the corresponding string value